src/components/home/constants.js

```diff
@@ -33,3 +27,23 @@ export const HOME_STRINGS = {
     WELCOME_TEXT_3: `Dear customer, we make sure you get a discount anytime you
         patronize us. Give us a try.`,
 };
+
+export const INPUTS = [
+    {
+        field: "name",
+        iconType: "user",
+        placeholder: "Enter your username",
+        rules: [{ message: "You have to input your username", required: true }],
+    },
+    {
+        field: "email",
+        iconType: "mail",
+        placeholder: "Please enter your email",
+    },
+    {
+        field: "subject",
+        placeholder: "Subject",
+    },
+];
+export const NAME = "home";
+export const CONTACT_URL = "http://localhost:4000/account/sendmail";
```

Comment:
`CONTACT_URL` should be `/account/sendmail`;

Reply:
good job
lgtm